### PR TITLE
Fix typos and complete and standardize the comments

### PR DESCRIPTION
**Pull request by Sagilio**
It fixed typos at `ManagementEnforcer.cs` and completed and standardized the comments

### CODE REVIEW

**Comment by hsluoyz** [comment]
@huazhikui @xcaptain @nodece please review.

File: NetCasbin/Rbac/IRoleManager.cs [hunk 1]
```diff
@@ -7,47 +7,43 @@ public interface IRoleManager
...
         /// 清除所有的数据并将角色管理器重置为初始状态
         /// </summary>
         void Clear();
 
         /// <summary>
-        /// addLink adds the inheritance link between two roles. role: name1 and role:
+        /// Adds the inheritance link between two roles. role: name1 and role:
         /// name2. domain is a prefix to the roles.
-        /// 
         /// 新增两个角色之间的继承关系
```
Comment (xcaptain) [inline NetCasbin/Rbac/IRoleManager.cs:17]: You may want to translate these chinese comments into English :smile:

File: NetCasbin/CoreEnforcer.cs [hunk 2]
```diff
@@ -16,108 +16,116 @@ namespace NetCasbin
     /// </summary>
     public class CoreEnforcer
     {
-        private IEffector eft;
+        private IEffector _effector;
```
Comment (xcaptain) [inline NetCasbin/CoreEnforcer.cs:19]: Like your coding style, you are using a _ for private member and lower case the first character for protected member right? You can also try dotnet-format to format the code.

File: NetCasbin/Rbac/IRoleManager.cs (hunk 1 above)
Comment (Sagilio) [reply NetCasbin/Rbac/IRoleManager.cs:17]: I found the English version is above at it..., It looks the Chinese version was translated from that.

File: NetCasbin/CoreEnforcer.cs (hunk 2 above)
Comment (Sagilio) [reply NetCasbin/CoreEnforcer.cs:19]: Yes. I will try it.

File: NetCasbin/Rbac/IRoleManager.cs (hunk 1 above)
Comment (hsluoyz) [reply NetCasbin/Rbac/IRoleManager.cs:17]: Remove all Chinese.

Comment (Sagilio) [reply NetCasbin/Rbac/IRoleManager.cs:17]: > Remove all Chinese.

Ok.

**Pull request merged**